test/e2e/util.go

```diff
@@ -2098,7 +2098,7 @@ func NewHostExecPodSpec(ns, name string) *api.Pod {
 	pod := &api.Pod{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: latest.GroupOrDie("").GroupVersion.Version,
```

Comment:
> To be extra clear, if mygrouppackage installs the codec & schema etc when imported, I would not be fine with that.

`pkg/apis/<group>/install` does exactly that today.  See here: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/install/install.go#L53 and here: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/install/install.go#L73 (scheme registered down in that method).  How were you thinking it was done?  `install` is different/special?